src/Helper.jl

```diff
@@ -26,6 +34,27 @@ function simulate!(cache::CachedEcosystem, srt::Unitful.Time, timestep::Unitful.
   cache.abundances.matrix[srt + timestep] = eco.abundances
 end
 
+"""
+    simulate!(eco::Ecosystem, times::Unitful.Time, timestep::Unitful.Time, cacheInterval::Unitful.Time, 
+cacheFolder::String, scenario_name::String)
+
+Function to run an ecosystem, `eco` for specified length of times, `duration`,
+for a particular timestep, 'timestep'. A cache interval and folder/file name 
+are specified for saving output.
+"""
+function simulate!(eco::Ecosystem, times::Unitful.Time, timestep::Unitful.Time, cacheInterval::Unitful.Time, 
+  cacheFolder::String, scenario_name::String)
+  time_seq = 0s:timestep:times
```

Comment:
I feel like this should be:
```suggestion
  time_seq = zero(times):timestep:times
```
in case it's floating point, though I'm also not sure that `times` is a great name for the argument - maybe `endtime`?